spec/javascripts/live_search_spec.js

```diff
@@ -161,21 +166,23 @@ describe("liveSearch", function(){
 
   describe('with relevant DOM nodes set', function(){
     beforeEach(function(){
+      GOVUK.analytics.trackEvent = function () {}
+
       liveSearch.$form = $form;
       liveSearch.$resultsBlock = $results;
       liveSearch.$countBlock = $count;
-      liveSearch.state = { field: "sheep" };
+      liveSearch.state = { field: "sheep", published_at: "2004" };
       liveSearch.$atomAutodiscoveryLink = $atomAutodiscoveryLink;
     });
 
     it("should update save state and update results when checkbox is changed", function(){
       var promise = jasmine.createSpyObj('promise', ['done']);
       spyOn(liveSearch, 'updateResults').and.returnValue(promise);
       //spyOn(liveSearch, 'pageTrack').and.returnValue(promise);
-      $form.find('input').prop('checked', false);
+      $form.find('input[name="field"]').prop('checked', false);
 
       liveSearch.formChange();
-      expect(liveSearch.state).toEqual([]);
+      expect(liveSearch.state).toEqual([{name: 'published_at', value: '2004'}]);
       expect(liveSearch.updateResults).toHaveBeenCalled();
       promise.done.calls.mostRecent().args[0]();
```

Comment:
I've checked and this does nothing (the function just returns the first argument for the call, which is undefined), it was added in 2014 and untouched since. Would you mind removing it and the stuff thats commented out